src/presenters/admin/migration-error-presenter.php

```diff
@@ -16,30 +17,53 @@
 class Migration_Error_Presenter extends Abstract_Presenter {
 
 	/**
-	 * Presents the warning that your site's content is not fully indexed.
+	 * Holds the migration error.
 	 *
-	 * @return string The warning HTML.
+	 * @param int|false $time    The timestamp.
+	 * @param string    $version The Yoast SEO version.
+	 * @param string    $message The error message.
+	 *
+	 * @var array
+	 */
+	protected $migration_error;
+
+	/**
+	 * Migration_Error_Presenter constructor.
+	 *
+	 * @param array $migration_error The migration error.
+	 */
+	public function __construct( $migration_error ) {
+		$this->migration_error = $migration_error;
+	}
+
+	/**
+	 * Presents the migration error that occurred.
+	 *
+	 * @return string The error HTML.
 	 */
 	public function present() {
-		$message = \sprintf(
+		$message    = \sprintf(
 			/* translators: %s: Yoast SEO. */
 			\esc_html__( '%s was unable to create the database tables required and as such will not function correctly.', 'wordpress-seo' ),
 			'Yoast SEO'
 		);
-		$support = \sprintf(
+		$support    = \sprintf(
 			/* translators: %1$s: link to help article about solving table issue. %2$s: is anchor closing. */
 			esc_html__( 'Please read %1$sthis help article%2$s to find out how to resolve this problem.', 'wordpress-seo' ),
 			'<a href="' . WPSEO_Shortlinker::get( 'https://yoa.st/3-6' ) . '">',
 			'</a>'
 		);
+		$debug_info = \sprintf(
+			'<details><summary>%1$s</summary><p>%2$s</p></details>',
+			\esc_html__( 'Show debug information', 'wordpress-seo' ),
+			\esc_html( $this->migration_error['message'] )
+		);
 
 		return \sprintf(
-			'<div class="notice notice-error">' .
-				'<p>%1$s</p>' .
-				'<p>%2$s</p>' .
-			'</div>',
+			'<div class="notice notice-error"><p>%1$s</p><p>%2$s</p>%3$s</div>',
```

Comment:
I find the original way of 'indenting' the code more readable:
```php
'<div class="notice notice-error">' .
	'<p>%1$s</p>' .
	'<p>%2$s</p>' .
    '%3$s' .
'</div>',
```

Reply:
I seem to remember that we should not concatenate things when not needed. And when possible just use `sprintf` instead. Which is why I changed this.